src/main/java/hudson/plugins/logparser/LogParserParser.java

```diff
@@ -69,8 +70,12 @@ public LogParserParser(final FilePath parsingRulesFile,
      * lists of links to these errors/warnings/info messages respectively :
      * errorLinks.html, warningLinks.html, infoLinks.html
      */
-    public LogParserResult parseLog(final AbstractBuild build)
-            throws IOException, InterruptedException {
+    @Deprecated
+    public LogParserResult parseLog(final AbstractBuild build) throws IOException, InterruptedException {
+        return this.parseLog((Run<?, ?>) build);
+    }
+
+    public LogParserResult parseLog(final Run<?, ?> build) throws IOException, InterruptedException {

```

Comment:
@oleg-nenashev We have followed this policy in Static Analysis Plugins. Anyway, I propose to remove all deprecated methods.This plugin needs a 2.0 version and release a new version as soon as possible.